server/controller/paymentController.js

```diff
@@ -25,3 +50,28 @@ exports.payment = async (req, res) => {
   }
 };
 
+exports.confirmPayment = async (req, res) => {
+  try {
+    const { session_id } = req.body;
+
+    const session = await stripe.checkout.sessions.retrieve(session_id);
+
+    if (session.payment_status === "paid") {
+      const bookingId = session.metadata.booking_id;
+
+      await Payment.findOneAndUpdate(
+        { booking_id: bookingId },
+        { status: "Completed", amount: session.amount_total / 100, payment_method: "Stripe" }
+      );
+
+      await Booking.findByIdAndUpdate(bookingId, { status: "Confirmed" });
+
+      res.status(200).json({ success: true });
+    } else {
+      res.status(400).json({ success: false, message: "Payment not completed" });
+    }
+  } catch (error) {
+    console.error("Error confirming payment:", error);
```

Comment:
please remove the console, its not required.

Reply:
i have removed the line